cartridges/app_storefront_job/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>app_storefront_job</name>
```

Comment:
Rename cartridge to 'app_custom_storefront', since this cartridge will contain all the custom features of the site.
Add all the custom code including pdp information to this cartridge.